easybuild/easyconfigs/h/HTSeq/HTSeq-0.9.1-foss-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = "PythonPackage"
+
+name = 'HTSeq'
+version = '0.9.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www-huber.embl.de/users/anders/HTSeq/'
+description = """A framework to process and analyze data from high-throughput sequencing (HTS) assays"""
+
+toolchain = {'name': 'foss', 'version': '2017b'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+checksums = ['af5bba775e3fb45ed4cde64c691ebef36b0bf7a86efd35c884ad0734c27ad485']
+
+dependencies = [
+    ('Python', '2.7.14'),
+    ('Pysam', '0.14', versionsuffix),
+    ('matplotlib', '2.1.0', versionsuffix),
+]
+
+use_pip = True
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': ["bin/htseq-count", "bin/htseq-qa"],
+    'dirs': ["lib/python%(pyshortver)s/site-packages/%(name)s-%(version)s-py%(pyshortver)s-linux-x86_64.egg"],
```

Comment:
The directory name is %(name)s-%(version)s-py%(pyshortver)s-linux-x86_64.egg-info
so either add "-info" at the end of that 'dirs' test
or we could test for just lib/python%(pyshortver)s/site-packages/%(name)s
which is also created.

The same for the other HTSeq easyconfigs.

Reply:
Just checking for `lib/python%(pyshortver)s/site-packages` is enough imho...

Reply:
I see the point - thank you for the information @boegel and @akesandgren !